### DISCUSSION TITLE
shot this on hero 11, 2.7k 120fps 4:3. I have gopro labs 200 bitrate activated, yet my data rate is only reading out at 44.66.. any ideas? I shot this on Delkin v90 64 gig card (picture attached) shouldn't this card be plenty quick enough for a faster bitrate?

### DISCUSSION THREAD

**Discussion by lukabees** [gopro/labs#418] [Q&A]
shot this on hero 11, 2.7k 120fps 4:3. I have gopro labs 200 bitrate activated, yet my data rate is only reading out at 44.66.. any ideas? I shot this on Delkin v90 64 gig card (picture attached) shouldn't this card be plenty quick enough for a faster bitrate?[](url)
<img width="598" alt="Screenshot 2023-06-19 at 2 15 08 PM" src="https://github.com/gopro/labs/assets/137114235/bc071fcd-08be-4325-938d-a1eb774e88e6">

**Comment by dnewman-gpsw** [comment]
Please share the metadata for that file using https://gopro.github.io/labs/control/metadata/

**Reply by dnewman-gpsw** [reply]
I tried this mode and is went to an increases data rate, so this not a camera/Labs bug.  I need to see the metadata to see what might be the issue.

**Reply by lukabees** [reply]
DEVC 
  DVID 1
  DVNM Global Settings
  VERS 8, 2, 2
  FMWR H22.01.02.01.70
  LINF LSU2081001402425
  CINF 94, 254, 150, 184, 53, 229, 135, 46, 210, 50, 60, 35, 196, 108, 91, 7
  CASN C3471325768070
  MINF HERO11 Black
  MUID 3096903262, 780657973, 591147730, 123432132, 0, 0, 0, 0
  CMOD 12
  MTYP 0
  OREN U
  DZOM Y
  DZST 0
  SMTR N
  PRTN Y
  PTWB 5500K
  PTSH MED
  PTCL FLAT
  EXPT AUTO
  PIMX 100
  PIMN 100
  PTEV 0.0
  RATE .empty.
  SROT 5.0000
  EISE N
  EISA N/A
  HCTL Off
  AUPT N
  APTO OFF
  AUDO AUTO
  BROD .empty.
  BRID 0
  PVUL F
  PRJT GPRO
  SOFF 0
  CLKS 2
  CDAT 0x0000000064452E3A
  SCTM 0
  PRNA 1
  PRNU 0
  SCAP N
  CDTM 0
  DUST NO_LIMIT
  VRES 2704, 2028
  VFPS 120000, 1001
  HSGT OFF
  BITR HIGH
  MMOD STEREO
  RAMP .empty.
  TZON 65056
  CLKC 0
  DZMX 1.4000
  CTRL Pro
  PWPR PERFORMANCE
  ORDP Y
  CLDP Y
  PIMD MANUAL
DEVC 
  DVID FOVL
  DVNM Large FOV
  ABSC 0.0000
  ZFOV 146.2798
  VFOV W
  MXCF x1
  MAPX 1.0000
  MYCF y1
  MAPY 1.0000
  PYCF r0r1r2r3r4r5r6
  POLY 0.0000, 1.9775, 0.1257, -0.8466, 0.4934, -0.0000, 0.0000
  ZMPL 0.7019
  ARUW 1.3333
  ARWA 1.3333
DEVC 
  DVID USRM
  DVNM User Metadata
  STRM 
    EMPT 2
    NR01 1
    EXPT 1
    64BT 1
    OWNR Luke Bredar\nREWARD\n
    BITR 200
    USRD 
DEVC 
  DVID HLMT
  DVNM Highlights

**Reply by dnewman-gpsw** [reply]
This is nothing looking wrong.  Other than the old firmware, we on 2.10.70 now. Try a !RESET!1O to clear out all the Labs settings, then only setting !MBITR=180 and re-test the capture.  Then add NR01, EXPT and OWNR if the bitrate is working. Avoid 200Mbs for high frame rates, as the captures are more likely to fail.  

P.S. Don't use 64BT and that is not needed on HERO11, large chapters are the default.

**Reply by lukabees** [reply]
ok thanks. how do I avoid 200mhz

**Reply by dnewman-gpsw** [reply]
By setting !MBITR=180 (i.e. lower than 200)

**Reply by lukabees** [reply]
is 180mbps safe for all framerates?

**Reply by dnewman-gpsw** [reply]
No, not for all cameras either. For 120fps try 160.  Labs is all about experimentation.
